src/main/java/tr/com/orioninc/laborant/controller/LabController.java

```diff
@@ -10,12 +11,9 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-import java.util.StringTokenizer;
-
+import java.util.*;
```

Comment:
Please avoid wildcard import statements, unless you're actually using all classes inside java.util

Reply:
done